.github/actions/setup/action.yml

```diff
@@ -13,11 +13,32 @@ runs:
             if [ ! -f "requirements.txt" ]; then
                 echo "python-gardenlinux-lib @ git+https://github.com/gardenlinux/python-gardenlinux-lib.git@${{ inputs.version }}" | tee -a requirements.txt
             fi
+        - name: Check for Python
+          id: python-check
+          run: |
+            if command -v python &/dev/null; then
+              echo "python_installed=true" >> $GITHUB_OUTPUT
+            else
+              echo "python_installed=false" >> $GITHUB_OUTPUT
+            fi
+        - name: Activate caching based on lockfile
```

Comment:
I think we can combine these two steps and name it `Evaluate Python environment`.

Reply:
Fixed it, thanks for the suggestion this makes it easier